prime-router/docs/playbooks/how-to-use-token-auth.md

```diff
@@ -0,0 +1,71 @@
+# How To Use FHIR Style Token Authentication and the Waters API.
```

Comment:
There is now a Smoke Test, 
`./prime test --run watersauth`
that does the same steps as in this hoot, plus more.